styles/layouts/SectionHeader.module.scss

```diff
@@ -0,0 +1,42 @@
+.sectionHeader {
+  @apply w-full;
+  @apply pt-4 px-4;
+  @apply flex;
+}
+
+.sectionHeaderContainer {
+  @apply rounded-t-lg;
+  @apply px-4 py-2;
+  @apply shrink;
+  @apply flex-grow-0;
+}
+
+.sectionHeaderTitle {
+  @apply text-2xl font-bold text-center text-color-2;
+}
+
+// Header Variants
+.style1 {
+  @apply bg-color-1;
+  & > span {
+    @apply bg-color-3;
+  }
+}
+.style2 {
+  @apply bg-color-3;
+  & > span {
+    @apply bg-color-1;
+  }
+}
+
+.rightAligned {
```

Comment:
Can we just add a comment above here something like `// Header Alignments`

Reply:
fixed